test/leveldown-substitution-test.js

```diff
@@ -7,14 +7,14 @@ var levelup = require('../lib/levelup.js')
 var assert = require('referee').assert
 var refute = require('referee').refute
 var buster = require('bustermove')
-var MemDOWN = require('memdown')
+var memdown = require('memdown')
 var encDown = require('encoding-down')
 
 require('./common')
 
 buster.testCase('LevelDOWN Substitution', {
   'test substitution of LevelDOWN with MemDOWN': function (done) {
-    var md = new MemDOWN('foo')
+    var md = memdown('foo')
```

Comment:
@ralphtheninja should be equivalent. I couldn't find a way to type memdown as function + newable + generic + static. It was a case of pick 3... but not all 4.